addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2189,6 +2207,14 @@ options.registry.HeaderNavbar = options.Class.extend({
                 return !weUtils.getCSSVariableValue('header-template').includes('hamburger');
             }
         }
+        if (widgetName === 'header_alignment_opt') {
+            if (!this.$target[0].querySelector('.o_offcanvas_menu_toggler')) {
+                // If mobile menu is "Default", hides the alignment option for
+                // "hamburger full" and "magazine" header templates.
+                return !["'hamburger-full'", "'magazine'"].includes(weUtils.getCSSVariableValue('header-template'));
```

Comment:
I also changed this line, the IDs still exist in V15 but I think it's more robust this way given the issue that occurred with the other line due to the forward-port.